cmd/edit-team-secret

```diff
@@ -13,6 +13,8 @@ TMP_FILE=$(mktemp)
 function cleanup()
 {
     rm $TMP_FILE
+    # try to remove backup files
+    rm $TMP_FILE* 2>1 || true
```

Comment:
`2>1` is redirecting stderr to a file named `1` :wink:

You probably meant `2>&1`, but I think `-f` is the way to go anyway (it [is posix](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/rm.html) after all):
> Do not prompt for confirmation. Do not write diagnostic messages or modify the exit status in the case of no file operands, or in the case of operands that do not exist.

```suggestion
    rm -f $TMP_FILE* || true
```

Reply:
And then we can probably also get rid of `|| true`, sorry to only notice now, will edit the suggestion above.